ui/reader/epub/src/commonMain/kotlin/com/kafka/reader/epub/ReaderScreen.kt

```diff
@@ -46,6 +46,7 @@ fun ReaderScreen(viewModel: ReaderViewModel) {
                 settingsState = settingsState,
                 tocState = tocState,
                 theme = state.settings.theme,
+                areActionButtonsEnabled = state.epubBook != null,
```

Comment:
compose changed the naming convention a bit. Notice material components have "enabled" parameter instead of "isEnabled", so `actionsIconsEnabled` would be a better name.

More importantly, I think the buttons should not show when loading. maybe we can hide them?